README.md

```diff
@@ -52,7 +52,6 @@ have enabled by default.
 To disable a rule, set its value to `null`.
```

Comment:
Nice, good to know! I'll fix the couple custom rules we have in this repo to allow for `false` and then update the doc.